roles/lib_openshift/src/doc/manage_node

```diff
@@ -0,0 +1,88 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oadm_manage_node
+short_description: Module to manage openshift nodes
+description:
+  - Manage openshift nodes programmatically.
+options:
+  kubeconfig:
+    description:
+    - The path for the kubeconfig file to use for authentication
+    required: false
+    default: /etc/origin/master/admin.kubeconfig
+    aliases: []
+  debug:
+    description:
+    - Turn on debug output.
+    required: false
+    default: False
+    aliases: []
+  node:
+    description:
+    - A list of the nodes being managed
+    required: false
+    default: None
+    aliases: []
+  selector:
+    description:
+    - The selector when filtering on node labels
+    required: false
+    default: str
```

Comment:
According to the Ansible Module `selector` has a default of `None` and type of `str`

> selector=dict(default=None, type='str')